control/statefbk.py

```diff
@@ -1079,14 +1046,14 @@ def ctrb(A, B, t=None):
     Parameters
```

Comment:
"Controllabilty" at the top of this docstring is now fixed on `main` branch via https://github.com/python-control/python-control/pull/1107